README.md

```diff
@@ -141,6 +148,33 @@ Validated Git signature: true
 Validated Rekor entry: true
 ```
 
+### Private Sigstore
+
+[Similar to cosign](https://docs.sigstore.dev/cosign/custom_components/),
+Gitsign can also run against other Sigstore instances besides the default public
+instance.
+
+There are three options to configure Gitsign to verify against custom
```

Comment:
Is there a reason to duplicate the content from the link rather than just say "see the link for how to do this"?

It feels like an Advanced Use Case (TM) and detracts from the succinctness of the current README

(won't push you super hard on this though)

Reply:
Done!